# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#feedback          当前样本个数 当前数据 当前平均值 当前标准差
int32 sample
float32 data
float32 mean
float32 std_dev
